Article/Article.js

```diff
@@ -1,26 +1,32 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
+    // console.log(this.domElement);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
Could also be written as `this.expandButton.addEventListener("click", () => this.expandArticle());`, no difference in what it does though, just preference.

Reply:
Ahh, I forgot about that one. thanks for pointing that out. Gotta get used to arrow functions !!